msteams-platform/includes/tabs/content-security-policy-headers.md

```diff
@@ -1,17 +1,6 @@
 | **Microsoft 365 app host** | **frame-ancestor permission** |
-|--|--|
-| *All hosts* (New) | `*.cloud.microsoft` |
+| --- | --- |
+| All hosts (New) | `*.cloud.microsoft` |
 | Teams | `teams.microsoft.com`, `*.teams.microsoft.com` |
 | Microsoft 365 app | `*.microsoft365.com`, `*.office.com`,  |
 | Outlook | `outlook.office.com`, `outlook.office365.com`, `outlook-sdf.office.com`, `outlook-sdf.office365.com` |
```

Comment:
@sbtron, could you please confirm if we need to add `*.copilot.com` here?

cc: @erikadoyle

Reply:
No copilot.com is not a Microsoft domain

Reply:
Thank you for confirming, @sbtron. Could you please approve the pull request on behalf of Chris?